### ISSUE TITLE
Distill burnable automation

### ISSUE THREAD

**Issue by EternalBlueFlame** [EternalBlueFlame/Traincraft-5#148]
I received a suggestion from a user (Gazer29) that the dilstil will not take in burnables piped to it, however it will take sugarcane.
Burnables should be added, along with empty diesel canisters if it isn't already supported.

**Comment by Staegrin** [comment]
Currently using version 4.3.0_003. You can hopper in petroleum, sugar cane, oak leaves and oil sands.  But you can put a hopper on the still and also get any other object into the top slot.

**Comment by 31trainman** [comment]
Piping in objects last I recall were not put into right slots unless that was fixed

**Comment by Bendragon1337** [comment]
How about the ability to have it RF-powered?

**Comment by NitroxydeX** [comment]
@Bendragon1337 that is against the idea behind Traincraft because we don't provide any cables. You would always need a second mod to play Traincraft which is bad. We would need to add a config option where you can enable it and redo the system of the Distill. That could be a thing for the rewrite but I'm not sure if it's really worth it

**Comment by Bendragon1337** [comment]
I don't agree that it ruins the idea behind Traincraft because there would still be the features to make it work in vanilla, though there could be optional "upgrades" similar to Progressive Automation to enable it to accept pipes.

I'm not sure if it's worth it either.
